source/install-desktops/modify-cluster-configuration.rst

```diff
@@ -0,0 +1,53 @@
+.. _install-desktops-modify-cluster-configuration:
+
+Modify Cluster Configuration
+============================
+
+If you haven't already done so, you will need to first have a corresponding
+cluster configuration file for the cluster you intend to launch desktops on. It
+is recommended you follow the directions on :ref:`add-cluster-config`.
```

Comment:
Mention that in order for this to work, job_adapter section in the cluster config needs to be filled out for the cluster.